packages/metal-soy/test/Soy.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 
+import { core } from 'metal';
```

Comment:
But more importantly... why are the tests here instead of in the `metal` package?